zp-relayer/utils/constants.ts

```diff
@@ -21,8 +21,10 @@ const constants = {
     maxTimeout: 60000,
     randomize: true,
   },
-  TRACE_ID: 'zkbob-support-id' as const,
   MESSAGE_PREFIX_COMMON_V1: '0000',
+  HEADER_TRACE_ID: 'zkbob-support-id' as const,
+  HEADER_LIBJS: 'zkbob-libjs-version' as const,
+  LIBJS_MIN_VERSION: '2.0.0',
```

Comment:
Maybe it's better to define min viersion in the environment variables?

Reply:
Don't think it makes sense since most often we need a specific version of relayer to accept requests from specific versions of library. But, for testing purposes, I would have a flag to disable the check of the library version in the header.